src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/aggregators/SlidingWindow.java

```diff
@@ -102,4 +102,5 @@ public double readSum() {
   public int size() {
     return windowDeque.size();
   }
+
```

Comment:
nit: can you remove this line, it isn't necessary for this change

Reply:
Removed.